### DISCUSSION TITLE
Adding stdbuf to nextcloud container

### DISCUSSION THREAD

**Discussion by dkellenb** [nextcloud/all-in-one#1531] [Ideas] [answered]
# Goal
As an administrator i'd like to enable notify for smb for faster change detection.

# Detail
Synchronisation can be improved by `occ files_external:notify <id>` . Unfortunatelly, starting the command results in ` stdbuf is required for usage of the notify command` .

# Expected change
Add `stdbuf`  package in ` nextcloud`  Docker Image.

**Comment by dkellenb** [comment]
Thanks to the documentation I have found a way how to add the dependency manually: ` NEXTCLOUD_ADDITIONAL_APKS=coreutils` . So for me it works.

**Comment by szaimen** [comment]
Hi, I agree that stdbuf should be added by default if it is required for files_external:notify. 

@relink2013 can you confirm that this package is required for the command?

**Reply by dkellenb** [reply]
The correct apk package is `coreutils`.

**Comment by szaimen** [comment, marked as answer]
Will be added with https://github.com/nextcloud/all-in-one/pull/1532

**Reply by szaimen** [reply]
This is now released with v4.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
